python/packages/autogen-core/src/autogen_core/logging.py

```diff
@@ -61,6 +61,95 @@ def __str__(self) -> str:
         return json.dumps(self.kwargs)
 
 
+class LLMStreamStartEvent:
+    """To be used by model clients to log the start of a stream.
+
+    Args:
+        messages (List[Dict[str, Any]]): The messages used in the call. Must be json serializable.
+
+    Example:
+
+        .. code-block:: python
+
+            from autogen_core import EVENT_LOGGER_NAME
+            from autogen_core.logging import
+
+            logger = logging.getLogger(EVENT_LOGGER_NAME)
+            logger.info(LLMStreamStartEvent())
```

Comment:
This sample code is also typed checked so we need to ensure the code can run.

Reply:
Which specific piece are you referring to? I added the `import` in, and also fixed the other examples to have all required args.

Reply:
> I added the import in, and also fixed the other examples to have all required args.

That's all I was referring to. I meant we have CI action that runs type checks on the code blocks in API doc.